src/lightning_app/core/app.py

```diff
@@ -64,6 +66,7 @@ def __init__(
         Arguments:
             root: The root LightningFlow component, that defines all the app's nested components, running infinitely.
                 It must define a `run()` method that the app can call.
+            flow_cloud_compute: The default Cloud Compute used for flow, Rest API and frontend's.
```

Comment:
there can be flows within root flows, so this on is only for root flow?
also wondering why is it not part of Flow construct?

Reply:
All flows always runs in the same process, on the same machine. Configuring different cloud compute for different flow components is not valid. Therefore, it gets defined on the app.